lib/core/form/components/widgets/container/container.widget.ts

```diff
@@ -57,6 +58,9 @@ export class ContainerWidgetComponent extends WidgetComponent implements OnInit,
     ngOnInit() {
         if (this.field) {
             this.content = new ContainerWidgetComponentModel(this.field);
+            this.numberOfColumns = (this.content.json?.numberOfColumns || 1) > (this.content.columns?.length || 1) ?
```

Comment:
can we create a method? is not easy to understand which check we do and why

Reply:
done thanks!